packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -172,11 +175,25 @@ export class FluidClientDebugger
 		["GET_CONTAINER_STATE"]: (untypedMessage) => {
 			const message = untypedMessage as GetContainerStateMessage;
 			if (message.data.containerId === this.containerId) {
+				console.log("Passed 1");
 				this.postContainerStateChange();
 				return true;
 			}
 			return false;
 		},
+		["AUDIENCE_EVENT"]: (untypedMessage) => {
```

Comment:
I don't think we need a listener for this event. The `AUDIENCE_EVENT` is an `outbound` event (i.e. one that is strictly posted by the debugger). 

I think the piece that we are probably missing is an inbound message requesting a complete summary of the audience data / history. That can be an `inbound` message the debugger listens for. Something like `GET_AUDIENCE_DATA`. And when the debugger receives that message, it can respond by posting the `AUDIENCE_EVENT`.

On the chrome extension side, when the audience view first launches, it can post the `GET_AUDIENCE_DATA` event, and wait for the response.

Reply:
Added `GET_AUDIENCE_EVENT` and created an event message for the event.